apps/api/v2/src/app.ts

```diff
@@ -17,10 +19,25 @@ export const bootstrap = (app: NestExpressApplication): NestExpressApplication =
     maxAge: 86_400,
   });
 
+  app.useGlobalPipes(
+    new ValidationPipe({
+      whitelist: true,
+      transform: true,
+      validationError: {
+        target: true,
+        value: true,
+      },
+      exceptionFactory(errors: ValidationError[]) {
+        return new BadRequestException({ errors });
+      },
+    })
+  );
+
   app.useGlobalFilters(new PrismaExceptionFilter());
   app.setGlobalPrefix("api", {
     exclude: [{ path: "health", method: RequestMethod.GET }],
   });
 
+  app.use(cookieParser());
```

Comment:
`suggestion-non-blocking`: Could consider adding `helmet`?

Reply:
Good idea, done in [feat: add helmet for security](https://github.com/calcom/cal.com/pull/12677/commits/57731a6ae07730394846da05b12632b64612245f)